app/classes/Transvision/ShowResults.php

```diff
@@ -315,20 +330,40 @@ public static function resultsTable($search_object, $search_results, $page)
                 $target_string2 = '';
             }
 
+            $string_id = md5($key . mt_rand());
+            $regular_string_id = 'string_' . $string_id;
+
+            // Find if we need to transliterate the string
```

Comment:
It would be nice to explain the logic of the following expression in the comment